regression-tests/sparktkregtests/lib/scoring_utils.py

```diff
@@ -33,21 +33,18 @@ def __init__(self, model_path, port_id, host=config.scoring_engine_host):
         self.name = host.split('.')[0]
         self.host = host
         # set port
-        config = SafeConfigParser()
+        port_config = SafeConfigParser()
         filepath = os.path.abspath(os.path.join(
-            os.path.dirname(os.path.realpath(__file__)),
-            "port.ini"))
-        config.read(filepath)
-        self.port = config.get('port', port_id)
+            config.root, "regression-tests",
+            "sparktkregtests", "lib", "port.ini"))
```

Comment:
minor: I think pep8 likes the args on either a single line, or each arg on individual lines.  Or at least I do.